configure

```diff
@@ -107,7 +109,6 @@ while true; do
   # Retry
 done
 
-
 # Find out where the CUDA toolkit is installed
 OSNAME=`uname -s`
 
```

Comment:
Ultimately we should simplify the part below so that only the user interaction remains in that script.